src/integrationTest/java/uk/gov/hmcts/reform/sendletter/tasks/FailedToPrintTaskTest.java

```diff
@@ -0,0 +1,185 @@
+    public void should_return_204_when_there_is_an_unprinted_letter() {
+        // given
+        Letter letter = createLetterAndReturn(secondBeforeCutOff);
+
+        ArgumentCaptor<Letter> captor = ArgumentCaptor.forClass(Letter.class);
+
+        // when
+        task.run();
+
+        // then
+        verify(insights).trackNotPrintedLetter(captor.capture());
+
+        // and
+        assertThat(captor.getAllValues()).hasSize(1);
+        assertThat(captor.getValue().getId()).isEqualByComparingTo(letter.getId());
+        assertThat(captor.getValue().getSentToPrintAt()).isEqualToIgnoringMillis(
+            Date.from(ZonedDateTime.now().minusDays(1).with(secondBeforeCutOff).toInstant())
+        );
+    }
+
+    @Test
+    public void should_not_pick_up_letter_if_sent_to_print_happened_after_the_deadline() {
+        // given
+        createLetterAndReturn(cutOff);
+
+        ArgumentCaptor<Letter> captor = ArgumentCaptor.forClass(Letter.class);
+
+        // when
+        task.run();
+
+        // then
+        verify(insights, never()).trackNotPrintedLetter(captor.capture());
+
+        // and
+        assertThat(captor.getAllValues()).isEmpty();
+    }
+
+    @Test
+    public void should_not_pick_up_letter_if_not_sent_to_print() {
+        // given
+        createLetterAndReturn(null);
+
+        ArgumentCaptor<Letter> captor = ArgumentCaptor.forClass(Letter.class);
+
+        // when
+        task.run();
+
+        // then
+        verify(insights, never()).trackNotPrintedLetter(captor.capture());
+
+        // and
+        assertThat(captor.getAllValues()).isEmpty();
+    }
+
+    @Test
+    public void should_not_pick_up_letter_if_it_is_marked_as_failed() {
+        // given
+        Letter letter = createLetterAndReturn(secondBeforeCutOff);
+        letter.hasFailed(true);
+
+        ArgumentCaptor<Letter> captor = ArgumentCaptor.forClass(Letter.class);
+
+        //when
+        repository.save(letter);
+        task.run();
+
+        // then
+        verify(insights, never()).trackNotPrintedLetter(captor.capture());
+
+        // and
+        assertThat(captor.getAllValues()).isEmpty();
+    }
+
+    @Test
+    public void should_not_pick_up_letter_if_it_is_already_printed() {
+        // given
+        Letter letter = createLetterAndReturn(secondBeforeCutOff);
+        letter.setState(LetterState.Posted);
+        letter.setPrintedAt(Timestamp.from(Instant.now()));
+
+        ArgumentCaptor<Letter> captor = ArgumentCaptor.forClass(Letter.class);
+
+        // when
+        repository.save(letter);
+        task.run();
+
+        // then
+        verify(insights, never()).trackNotPrintedLetter(captor.capture());
+
+        // and
+        assertThat(captor.getAllValues()).isEmpty();
+    }
+
+    private Letter createLetterAndReturn(LocalTime withSentToPrintAt) {
+        Letter letter = new Letter(MessageIdProvider.randomMessageId(), "service", null, "type", null);
+
+        if (withSentToPrintAt != null) {
```

Comment:
it's never null in tests

Reply:
there must be one test case (first one iirc) where i just create the letter